styles.css

```diff
@@ -0,0 +1,429 @@
+/* =============== FONT =============== */
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap');
+
+/* #093b5f    dark-blue*/
+/* #8bc34a    green*/
+/* #ebebeb    light-grey*/
+
+/* =============== GLOBAL =============== */
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  overflow-x: hidden;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  color: #093b5f;
+}
+
+h2 {
+  padding: 2rem 0;
+}
+
+p {
+  color: #555;
+  line-height: 1.5;
+}
+
+.btn {
+  background-color: #8bc34a;
+  text-align: center;
+  display: inline-block;
+  padding: .5rem 2rem;
+  border-radius: 3px;
+  text-decoration: none;
+  color: #fff;
+  cursor: pointer;
+  transition: all .2s ease-in-out;
+}
+
+.btn:hover {
+  background-color: #6f9c3c;
+}
+
+.container {
+  max-width: 860px;
+  margin: 0 auto;
+}
+
+/* =============== HEADER =============== */
+
+header {
+  height: 25rem;
+  color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+header::before {
+  content: "";
+  position: absolute;
+  left: 0;
+  height: 25rem;
+  width: 100vw;
+  background: url('images/header-image.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  z-index: -20;
+}
+
+.logo-header {
+  width: 50%;
+}
+
+.logo-header,
+.checkout-wrapper {
+  padding-top: 3rem;
+}
+
+header p {
+  color: #8bc34a;
+  font-size: 1.5rem;
+  font-style: italic;
+  position: relative;
+  top: -14px
```

Comment:
Make sure to update px to responsive units: %, em, rem. px are only acceptable for border line width.

Reply:
Updated. Thanks.